deploy/examples/import-external-cluster.sh

```diff
@@ -167,6 +168,20 @@ function importConfigMap() {
   fi
 }
 
+function createInputCommadConfigMap() {
+  if ! $KUBECTL -n "$NAMESPACE" get configmap "$EXTERNAL_COMMAND_CONFIGMAP_NAME" &>/dev/null; then
+    $KUBECTL -n "$NAMESPACE" \
+      create \
+      configmap \
+      "$EXTERNAL_COMMAND_CONFIGMAP_NAME" \
+      --from-literal=command="$EXTERNAL_CLUSTER_USER_COMMAND"
+  else
+    echo "configmap $EXTERNAL_COMMAND_CONFIGMAP_NAME already exists, update it"
```

Comment:
Users may interpret "update it" to mean that they need to take action. 

```suggestion
    echo "configmap $EXTERNAL_COMMAND_CONFIGMAP_NAME already exists, updating it"
```